src/components/LabelledData/index.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+
+const defaultProps = {
+  html: false,
+};
+
+const inputProps = {
+  label: PropTypes.oneOfType([PropTypes.string, PropTypes.element]).isRequired,
+  value: PropTypes.oneOfType([PropTypes.string, PropTypes.element]).isRequired,
+  html: PropTypes.bool,
+};
+
+function LabelledData({ label, value, html }) {
+  let valueDiv;
+
+  if (html) {
+    valueDiv = (
+      <div
+        className="labelled-data-value col"
+        dangerouslySetInnerHTML={{ __html: value }} // eslint-disable-line react/no-danger
```

Comment:
Yah, that's how React makes you set any text that you want to be parsed & rendered as HTML. So some fields from Discovery like Syllabus or Outcomes is given as raw HTML. Only way to get it on page is this way. It's ominious because they want you to think hard before you do it.

In our case, this should be fairly safe since not just anyone can edit that data.